getblocklab/block-lab
# Ask BL users to migrate to GCB

#### Changes
* This notice will supersede the previous notice.
* The previous notice will never display.
* But even if they've dismissed the previous notice, this will show.
* Still, this will only show on specific pages, not on every `/wp-admin` page.


#### Testing instructions
1. `composer install && npm install && npm run dev`
2. Go to `/wp-admin` > Block Lab
3. Expected: the new notice appears ![here-are-content-blocks](https://user-images.githubusercontent.com/4063887/112499876-7fbca600-8d4d-11eb-908f-2c0b97a9d5ce.png)
4. Click 'Migrate'
5. Expected: the new migration copy and `.gif` appear:
![Screen Shot 2021-03-25 at 9 57 37 AM](https://user-images.githubusercontent.com/4063887/112503513-99132180-8d50-11eb-9b14-ea0e424d8a92.png)
